pkg/bootstrap/docker/up.go

```diff
@@ -511,6 +519,31 @@ func (c *ClientStartConfig) EnsureHostDirectories(io.Writer) error {
 	return c.HostHelper().EnsureVolumeShare()
 }
 
+// EnsureDefaultRedirectURIs merges a default URL to an auth client's RedirectURIs array
+func (c *ClientStartConfig) EnsureDefaultRedirectURIs(out io.Writer) error {
+	f, err := c.Factory()
+	if err != nil {
+		return err
+	}
+
+	patch := fmt.Sprintf("{%q:[%q]}", "redirectURIs", defaultRedirectURI)
```

Comment:
@csrwng 
> actually, why not use the go client to make the call?

Hm, I am not sure what you mean. I thought `NewCmdPatch` was already using the client to make the remote call?